tests/php/LeftAndMainTest.php

```diff
@@ -59,7 +59,7 @@ public function tearDown()
     public function testExtraCssAndJavascript()
     {
         $admin = $this->objFromFixture(Member::class, 'admin');
-        $this->session()->set('loggedInAs', $admin->ID);
+        $this->logInAs($admin);
```

Comment:
Use `$this->logInWithPermission('ADMIN');` or `$this->logInAs('amin');`

Reply:
It's using a specific fixture, so it's correct as is

Reply:
When you give a string to [`SapphireTest::logInAs`](
https://github.com/silverstripe/silverstripe-framework/blob/dcdc25500be729f4340d42f8a4fc797461f862f3/src/Dev/SapphireTest.php#L1164-L1172), it will assumes it's a Member fixture ID and fetch it for you. Unless there's an actual need for having an actual `Member` instance in your test, calling `$this->idFromFixture(Member::class, 'admin')` or `$this->objFromFixture(Member::class, 'admin')` just adds pointless noise to your test and makes it harder to understand.

Reply:
Updated